uber/automated_emails.py

```diff
@@ -159,15 +257,18 @@ def days_before(days, dt, until=None):
                  needs_approval=False)
 
 MarketplaceEmail('Your {EVENT_NAME} Dealer registration is due in one week', 'dealers/payment_reminder.txt',
-                 lambda g: g.status == c.APPROVED and days_before(7, c.DEALER_PAYMENT_DUE, 2) and g.is_unpaid,
+                 lambda g: g.status == c.APPROVED and g.is_unpaid,
+                 date_filters=days_before(7, c.DEALER_PAYMENT_DUE, 2),
```

Comment:
Can I suggest ``when`` as a parameter name instead of ``date_filters``?  I like that a lot better for three reasons:

1) It's shorter.  Kind of dumb but definitely a preference.

2) ``when=days_before(7, c.DEALER_PAYMENT_DUE)`` reads very naturally and seems just as if not more descriptive.

3) Though I strongly support the use of ``listify`` to allow us to pass either a single filter or multiple, it's awkward for the parameter name to be pluralized but for the argument to be singular.  Calling it ``when`` gets around this problem.